### PR TITLE
Update dotnet. Add sudo to alpine

### PR DESCRIPTION
**Pull request by eivantsov**
### What does this PR do?
Update dotnet to 2.0 release
Adds sudo to alpine image
### What issues does this PR fix or reference?
https://github.com/eclipse/che-dockerfiles/issues/121
https://github.com/eclipse/che-dockerfiles/pull/123

### CODE REVIEW

File: recipes/dotnet_core/Dockerfile [hunk 1]
```diff
@@ -7,17 +7,19 @@
...
+    sudo mv ~/microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg && \
+    sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-xenial-prod xenial main" > /etc/apt/sources.list.d/dotnetdev.list' && \
     wget -qO- https://deb.nodesource.com/setup_7.x | sudo -E bash - && \
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF && \
     echo "deb http://download.mono-project.com/repo/ubuntu beta-xenial main" | sudo tee /etc/apt/sources.list.d/mono-official-beta.list && \
-    sudo apt-get update && \
-    sudo apt-get install -y \
-    dotnet-sdk-2.0.0-preview2-006497 \
+    sudo apt-get update
+RUN    sudo apt-get install -y \
```
Comment (benoitf) [inline recipes/dotnet_core/Dockerfile]: could you say why you've added a new RUN instruction ?

Comment (eivantsov) [inline recipes/dotnet_core/Dockerfile]: Build failed without `apt-transport-https -y`

Comment (benoitf) [inline recipes/dotnet_core/Dockerfile]: I mean, a separate RUN instruction instead of continuing the previous `RUN`

Comment (eivantsov) [inline recipes/dotnet_core/Dockerfile]: Ah, yes. Fixed.

Comment (benoitf) [inline recipes/dotnet_core/Dockerfile]: you still have another new RUN instruction 
`RUN    sudo apt-get install -y \`

after `sudo apt-get update`

**Pull request merged**